test/persist/test_failure_indices.sh

```diff
@@ -0,0 +1,124 @@
+#! /bin/bash
+
+export PATH="$GOBIN:$PATH"
+export TMHOME=$HOME/.tendermint_persist
+
+rm -rf "$TMHOME"
+tendermint init
+
+# use a unix socket so we can remove it
+RPC_ADDR="$(pwd)/rpc.sock"
+
+TM_CMD="tendermint node --log_level=debug --rpc.laddr=unix://$RPC_ADDR" # &> tendermint_${name}.log"
```

Comment:
With multiple arguments, this would probably be better as a shell function, to save quoting issues further along.
```suggestion
tm_node() {
   local logfile="$1"
   tendermint node --log_level=debug --rpc.laddr="unix://$RPC_ADDR" &> "$logfile"
}
```